### PR TITLE
[TELCODOCS-1385, TELCODOCS-1506] Telco core and RAN DU RDS consolidated PR

### PR DESCRIPTION
**Pull request by aireilly**
This PR consolidates the following approved docs PRs:

https://github.com/openshift/openshift-docs/pull/64278
https://github.com/openshift/openshift-docs/pull/65732

Issue:
https://issues.redhat.com/browse/TELCODOCS-1385
https://issues.redhat.com/browse/TELCODOCS-1506

Merge to main, CP to enterprise-4.14+

Preview:
https://file.emea.redhat.com/aireilly/rds-final/telco_ref_design_specs/telco-ref-design-specs-overview.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/telco-core-rds-other.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/core/telco-core-ref-components.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-other{context}"]
+= Other
```
Comment (imiller0) [inline modules/telco-core-rds-other.adoc:7]: Maybe Additional configuration?

File: modules/ztp-telco-ran-software-versions.adoc [hunk 2]
```diff
@@ -1,25 +1,26 @@
 // Module included in the following assemblies:
 //
 // * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
+// * telco_ref_design_specs/ran/telco-ran-ref-software-artifacts.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="ztp-telco-ran-software-versions_{context}"]
-= Telco RAN {product-version} validated software components
+= Telco core and RAN DU {product-version} validated software components
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/ztp-telco-ran-software-versions.adoc:8]: Yes I had wondered about that. Cool will update.

File: telco_ref_design_specs/core/telco-core-rds-use-cases.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+.Telco core use model architecture
+image:473_OpenShift_Telco_Core_Reference_arch_1123.png[Use model architecture]
+
+The networking prerequisites for {rds} functions are diverse and encompass an array of networking attributes and performance benchmarks. IPv6 is mandatory, with dual-stack configurations being prevalent. Certain functions demand maximum throughput and transaction rates, necessitating user plane networking support such as DPDK. Other functions adhere to conventional cloud-native patterns and can use solutions such as OVN-K, kernel networking, and load balancing.
+
+{rds} clusters are configured as standard three control plane clusters with worker nodes configured with the stock non real-time (RT) kernel. To support workloads with varying networking and performance requirements, worker nodes are segmented using `MachineConfigPool` CRs. For example, this is done to separate non-user data plane nodes from high-throughput nodes. To support the required telco operational features, the clusters have a standard set of Operator Lifecycle Manager (OLM) Day 2 Operators installed.
```
[the remark this answers is not in the document]
Comment (aireilly) [reply telco_ref_design_specs/core/telco-core-rds-use-cases.adoc:17]: Updated. Searched for `^{rds}`

**Comment by imiller0** [comment]
LGTM

**Pull request merged**